crates/codegen/parser_runtime/src/napi/napi_parse_error.rs

```diff
@@ -0,0 +1,35 @@
+use napi_derive::napi;
+
+use super::*;
+use napi_text_index::*;
+
+#[napi(namespace = "parse_error")]
+#[derive(PartialEq, Clone)]
+pub struct ParseError(RustParseError);
```

Comment:
I wonder why the change to write the additional type, instead of using `$[cfg_attr]` on the original one directly?
This way, all types/APIs can evolve together, and we don't have to maintain separate heirarchies.